api/v1alpha1/networkchaos_webhook.go

```diff
@@ -281,5 +281,9 @@ func (in *BandwidthSpec) validateBandwidth(bandwidth *field.Path) field.ErrorLis
 
 // validateTarget validates the target
 func (in *Target) validateTarget(target *field.Path) field.ErrorList {
+	if in.TargetMode != OnePodMode && in.TargetMode != AllPodMode && in.TargetMode != FixedPodMode && in.TargetMode != FixedPercentPodMode && in.TargetMode != RandomMaxPercentPodMode {
```

Comment:
Is `TargetMode` field required？

If so, we can modify networkchaos's validation.

https://github.com/chaos-mesh/chaos-mesh/blob/8ce400a4eb83ece39dbd28d4f5a013850bbf9f84/api/v1alpha1/networkchaos_types.go#L89-L90

Change it to
```
// +kubebuilder:validation:Enum=one;all;fixed;fixed-percent;random-max-percent
```

Reply:
@Gallardot  If the `Target` is not empty, `TargetMode` is required.